### DISCUSSION TITLE
I'm noob in dong contribution, pull request, etc, so apolagize now asking..

### DISCUSSION THREAD

**Discussion by abdulbadii** [vlang/v#20259] [Questions and Answers]
Please one clarify really clear yet simple how one correctly contribute PDF of the V documentation and its creation automation,
as I have had one and its script to automate it by small node lib., puppeteer, requirement

not only am i noob on this such , but also in many github's ways and rules to work efficiently, so now this isn't in github related facility but
here
https://drive.google.com/file/d/1A5o2L2fUO5ix64SIVlgEzqOZZO_8ryx8/view?usp=drive_link
https://drive.google.com/file/d/10Yg_DcMNcEooeLSvXetgVvZNViCX7zqV/view?usp=drive_link

Please help sincerely

**Comment by medvednikov** [comment]
Hi @abdulbadii 

You can easily do that with a tool like html2pdf
